database-seed/database-schema.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+const productImageScheme = mongoose.Schema({
```

Comment:
I'm guessing you wanted this to be `productImageSchema` but maybe not!

Reply:
Good catch, I'll clean that up